test/lib/routes/index.test.js

```diff
@@ -140,7 +140,7 @@ describe('Application routes', function () {
       post.end(new Buffer(JSON.stringify(data)));
 
       app.construct.once('store', function store(spec, progress, resData) {
-        const file = resData.files.files[0];
+        const file = resData.files.files.find(file => file.filename === 'index.min.jsx');
```

Comment:
Yes. And sure we can but this is duplicate work as I have to do it in the workers-factory library.